<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WndDevices</class>
 <widget class="QMainWindow" name="WndDevices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deskemes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="devicesScroller">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="devicesContainer">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>780</width>
         <height>539</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="lblNoDevices">
          <property name="text">
           <string>No devices</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Device">
    <property name="title">
     <string>&amp;Device</string>
    </property>
    <addaction name="actDeviceNew"/>
    <addaction name="separator"/>
    <addaction name="actExit"/>
   </widget>
   <widget class="QMenu" name="menu_Message">
    <property name="title">
     <string>&amp;Message</string>
    </property>
    <addaction name="actMessageSendNew"/>
   </widget>
   <addaction name="menu_Device"/>
   <addaction name="menu_Message"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actDeviceNew">
   <property name="text">
    <string>Add &amp;new...</string>
   </property>
   <property name="toolTip">
    <string>Search for and pair with a new device</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="actMessageSendNew">
   <property name="text">
    <string>&amp;Send new...</string>
   </property>
   <property name="toolTip">
    <string>Send a new text message</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
